content/en/blog/_posts/2020-12-08-kubernetes-release-1.20.md

```diff
@@ -0,0 +1,145 @@
+---
+layout: blog
+title: 'Kubernetes 1.20: The Raddest Release'
+date: 2020-12-08
+slug: kubernetes-1-20-release-announcement
+---
+
+**Authors:** [Kubernetes 1.20 Release Team](https://github.com/kubernetes/sig-release/blob/master/releases/release-1.20/release_team.md)
+
+We’re pleased to announce the release of Kubernetes 1.20, our third and final release of 2020! This release consists of 42 enhancements: 11 enhancements have graduated to stable, 15 enhancements are moving to beta, and 16 enhancements are entering alpha.
+
+The 1.20 release cycle returned to its normal cadence of 11 weeks following the previous extended release cycle. This is one of the most feature dense releases in a while: the Kubernetes' innovation cycle is still trending upward. This release has more Alpha vs Stable enhancements, showing that there is still a lot to be explored in the cloud native ecosystem.
+
+## Major Themes
+
+### Volume Snapshot Operations Goes Stable
```

Comment:
We usually write headings in sentence case, but for blog articles it's less important to get this right I think.

Reply:
I referenced previous blogs to try and find a pattern. We should probably come up with a standard.